web/cobrands/fixmystreet/assets.js

```diff
@@ -360,6 +349,9 @@ function select_nearest_asset() {
     var nearest_feature = this.getNearestFeature(marker.geometry, threshold);
     if (nearest_feature) {
         get_select_control(this).select(nearest_feature);
+    } else if (this.fixmystreet.disable_pin_snapping) {
```

Comment:
Not sure this fixes the issue, as this is only run on layer load isn't it? So what clears the fields if you click elsewhere on the map?